sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityClientOptions.java

```diff
@@ -26,7 +27,9 @@
      * Creates an instance of IdentityClientOptions with default settings.
      */
     public IdentityClientOptions() {
-        authorityHost = DEFAULT_AUTHORITY_HOST;
+        Configuration configuration = Configuration.getGlobalConfiguration().clone();
+        authorityHost = configuration.contains(configuration.PROPERTY_AZURE_AUTHORITY_HOST)
+        ? configuration.get(configuration.PROPERTY_AZURE_AUTHORITY_HOST) : DEFAULT_AUTHORITY_HOST;
```

Comment:
`Configuration` has a `T get(String configurationName, T defaultValue)` method that could simplify this logic.

```java
authorityHost = configuration.get(Configuration.PROPERTY_AZURE_AUTHORITY_HOST, DEFAULT_AUTHORITY_HOST);
```